knative-serving/config-applications/configuring-kourier.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
```

Comment:
It seems to follow this. Note also that since Serverless docs are now published separately and tracked inside their own branch (with other OpenShift docs files absent), we have some differences from the `main` `openshift-docs`.